### PR TITLE
docs: broken link in theme/inheritance docs

### PR DESCRIPTION
**Pull request by saintmalik**
**Summary**

Easy navigation and easy reading for others

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by saintmalik** [comment]
@billyyyyy3320 can you review this

File: packages/docs/docs/theme/inheritance.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We have two main reasons to support this feature:
 
 1. VuePress provides users with a [default theme](./default-theme-config.md), which meets the needs of document writers in most scenarios, even so, there are still many users who choose to `eject` and edit it, even if they may only need to make minor changes to one of the components.
 
-2. In [0.x](https://vuepress.vuejs.org/guide/custom-themes.html#site-and-page-metadata), only one `Layout.vue` is needed for a theme, so we can achieve simple expansion by directly wrapping `Layout.vue` of another theme.
+2. In [0.x](https://web.archive.org/web/20180915084828/https://vuepress.vuejs.org/guide/custom-themes.html), only one `Layout.vue` is needed for a theme, so we can achieve simple expansion by directly wrapping `Layout.vue` of another theme.
```
Comment (billyyyyy3320) [inline packages/docs/docs/theme/inheritance.md:9]: `https://v0.vuepress.vuejs.org/guide/custom-themes.html` would be better

Comment (saintmalik) [reply packages/docs/docs/theme/inheritance.md:9]: Will make the changes now

Comment (saintmalik) [reply packages/docs/docs/theme/inheritance.md:9]: can you please apply the fix, i cant make the edits from here

**Comment by saintmalik** [comment]
closing this

**Pull request closed without merging**